### ISSUE TITLE
keepDirtyOnReinitialize

### ISSUE THREAD

**Issue by amotavasseli** [formik/formik#2590]
Hi, It looks like you removed keepDirtyOnReinitialize from the repo. Is there another option that will handle this? https://github.com/formik/formik/pull/181/commits/75b43ae602804ea1e2113b6c2b9032193cabc5e3

**Comment by rayhero** [comment]
yes I also need this feature, why it's not include in latest version?

**Comment by RadoslavK** [comment]
Any workarounds?
